news/index.html

```diff
@@ -2,9 +2,11 @@
 title: News
 layout: page
 description: Latest CivicActions Accessibility news.
+redirect_from:
+- /posts/
```

Comment:
This fixes the breadcrumb link for posts. Also the remark-link in this file can be ignored.